src/client/client/game.js

```diff
@@ -263,18 +289,30 @@ let game = function() {
 			_enteredMarbleList[marble.entryId] = marble;
 
 			// Add mesh
-			marbleManager.spawnMarble(marble);
+			let mesh = marbleManager.spawnMarble(marble);
 
 			// Add UI stuff
 			let listEntry = _DOMElements.marbleListTemplate.cloneNode(true);
 			listEntry.removeAttribute("id");
+			listEntry.getElementsByClassName("camera")[0].addEventListener("click", function() {
+				_toggleMarbleTracking(marble, mesh);
+			}, false);
+			if (marble.finished) listEntry.classList.add("finished");
+			listEntry.getElementsByClassName("name")[0].innerText = marble.name;
 			listEntry.getElementsByClassName("name")[0].innerText = marble.name;
 			listEntry.getElementsByClassName("color")[0].style.background = marble.color;
-			listEntry.getElementsByClassName("time")[0].innerText = marble.time ? `🏁 ${(marble.time * .001).toFixed(2)}s` : "";
+			listEntry.getElementsByClassName("time")[0].innerText = marble.time ? `${(marble.time * .001).toFixed(2)}s` : "";
 			listEntry.getElementsByClassName("rank")[0].innerText = !isNaN(marble.rank) && marble.rank !== null ? `#${marble.rank + 1}` : "";
 			listEntry.style.order = marble.rank;
 			_enteredMarbleList[marble.entryId].listEntryElement = listEntry;
 
+			if (_userData && _userData.username === marble.name) {
```

Comment:
Checking for `_userData.id === marble.userId` is probably the safer option here.